packages/utils/src/testUtilities/createJestMockInstance.js

```diff
@@ -3,7 +3,6 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { jest } from '@jest/globals';
```

Comment:
So where does it get `jest` from below?

Reply:
It's global during tests! Same with `expect`, `describe` etc

Reply:
I knew that but I thought explicitly importing `jest` would be a better practice. But `jest` complains, it seems you can only do that in test files, which kind of beats the purpose 🤔

Reply:
I guess they want to discourage people from loading `jest` from `@jest/globals` in a non test environment and expect it'll work (it won't)